### ISSUE TITLE
Why all the parameters come to NaN when using small batchSize?

### ISSUE THREAD

**Issue by jinluyang** [vlfeat/matconvnet#547]
I've been training on the ORL face database for face recognition use.I tried different batchSize and found that when using batchSize smallerthan 40,all the weights of the net turn into NaN after some epochs.the objective curve stops drawing and the error curve stays at 0.975 and 0.875.Can anyone tell me why?

**Comment by jinluyang** [comment]
I just found that when using a large learning rate,the same thing happens.

**Comment by hyassine** [comment]
Hi jinluyang

Which classifier are you using ?

Regards

**Comment by jinluyang** [comment]
@hyassine vl_nnconv  It was about a year ago .I don't know whether the newest version has the same problem.

**Comment by hyassine** [comment]
@jinluyang  would you mind send me your scripts (and the version of matconvnet used) or publish them to debug the issue ?

**Comment by jinluyang** [comment]
@hyassine I just tried the code and found that the problem doesn't occur no matter what batchsize I use. So I think maybe I should close this issue.
